miniircd

```diff
@@ -1222,5 +1222,9 @@ def main(argv: Sequence[str]) -> None:
             server.remove_pid_file(args.pid_file)
 
 
+def console_script_entry_point():
```

Comment:
The tests now fail:

```
% ./test
miniircd:1225: error: Function is missing a return type annotation
miniircd:1230: error: Call to untyped function "console_script_entry_point" in typed context
Found 2 errors in 1 file (checked 2 source files)
```

That can be fixed by adding a return type annotation, but I would suggest using `main` as the entrypoint instead and just remove the `argv` parameter which actually is unused since 93297603fa70423f2c83f58d3dd23435a277e322.